charging_stations_pipelines/pipelines/osm/osm_mapper.py

```diff
@@ -1,23 +1,45 @@
+"""This module contains the methods for mapping the OpenStreetMap (OSM) data to the database models."""
 import logging
+import math
 from datetime import datetime
-from typing import Dict, Optional
+from typing import Any, Final, Optional
 
 from geoalchemy2.shape import from_shape
 from shapely.geometry import Point
 
 from charging_stations_pipelines.models.address import Address
 from charging_stations_pipelines.models.charging import Charging
 from charging_stations_pipelines.models.station import Station
-from charging_stations_pipelines.pipelines.shared import check_coordinates
+from charging_stations_pipelines.pipelines.osm import DATA_SOURCE_KEY
+from charging_stations_pipelines.pipelines.shared import check_coordinates, lst_flatten
+from charging_stations_pipelines.shared import filter_none, try_clean_str, try_float, try_split_str
 
 logger = logging.getLogger(__name__)
 
+SOCKET_TYPES = {  # type: Final[dict[str, str]]
+    'socket:chademo':            'CHAdeMO',
+    'socket:schuko':             'AC Schuko',
+    'socket:tesla_supercharger': 'Tesla Supercharger',
+    'socket:type2':              'AC Steckdose Typ 2',
+    'socket:type2:combo':        'AC Steckdose Typ 2 Combo',
+    'socket:type2_combo':        'AC Steckdose Typ 2 Combo',
```

Comment:
Socket Type2 Combo is not an AC Steckdose. From my point of view it has to be mapped to CCS Typ 2 which is capable of AC and DC

Reply:
![image](https://github.com/comsysto/eCharm/assets/49154842/6ad017bf-37d2-4d65-9960-3f0132e0e9b5)

Reply:
Fixed.